controllers/gitopsservice_controller_test.go

```diff
@@ -105,19 +105,17 @@ func TestReconcileDisableDefault(t *testing.T) {
 
 	var err error
 
-	fakeClient := fake.NewFakeClient(newGitopsService())
+	fakeClient := fake.NewClientBuilder().WithRuntimeObjects(newGitopsService()).Build()
```

Comment:
@anandf what was the motivation behind this and other similar changes?

Reply:
Reverted the changes. I will submit a separate PR for fixing the deprecated method calls.